packages/app/src/utils/process.ts

```diff
@@ -0,0 +1,3 @@
+export function getHeaderId(location: string): string {
+  return location.split('?id=eq.')[1];
```

Comment:
It would be just a little safer to `try {} catch () {}` that with an error handling.

I admit I should have created a global error handler and I didn't but it would not be so complicated to create a new one using the same principle I use in sagas with `libs/toast.js`: https://github.com/SocialGouv/code-du-travail-backoffice/blob/master/packages/app/src/sagas/answers/cancel.js#L34

In an ideal world, it should be thrown via a Redux action able to handle all kind of errors but there is no need to go that way since you will surely give up on Redux later on.

Reply:
To my mind, we don't need to set up any kind of trigger if this function failed. We have to make unit test, to test it. Thus, I `created` a `try - catch`